packages/runner/src/scanner/page.ts

```diff
@@ -28,12 +28,17 @@ export class Page {
     }
 
     public async goto(url: string): Promise<void> {
-        const gotoUrlPromise = this.puppeteerPage.goto(url, { waitUntil: ['load'] });
+        const gotoUrlPromise = this.puppeteerPage
+            .goto(url, { waitUntil: ['load'] })
+            .then(response => {
+                if (response.headers()['content-type'] !== 'text/html ') {
+                    throw Error('unsupported page type');
+                }
+            });
         const waitForNetworkLoadPromise = this.puppeteerPage.waitForNavigation({ waitUntil: ['networkidle0'], timeout: 15000 });
 
-        await gotoUrlPromise;
-
         try {
+            await gotoUrlPromise;
```

Comment:
will this try{} ignore above throw for content-type?

Reply:
fixed